### DISCUSSION TITLE
Visualizing TF accessibility for eRegulons

### DISCUSSION THREAD

**Discussion by tu4n-p** [aertslab/scenicplus#74] [Q&A] [answered]
Hi, I was wondering if there's anyway I could visualize TF accessibility ( just like what the software did for TF expression/Target gene expression/region accessibility with UMAP/tSNE) for eRegulons of interest? 
Thank you!

**Comment by SeppeDeWinter** [comment]
Hi @tu4n-p 

What do you exactly mean with TF-accessibility? You can visualise the accessibility of the target regions using the AUC-values, but I don't think this is what you're looking for?

Best,

Seppe

**Comment by tu4n-p** [comment]
Hi @SeppeDeWinter , sorry about the confusion. I guess what I meant is TF gene accessibility/differentially accessible TF genes, not the target genes . In the human cerebrium tutorial, it plotted something like this. <img width="924" alt="Screen Shot 2022-12-19 at 9 19 20 AM" src="https://user-images.githubusercontent.com/76016401/208446685-311a9c5c-07c3-42a3-b16a-860138513e01.png"> So I was wondering if I can also plot PRRX_2 accessibility on the same UMAP as well.
Thanks,

**Comment by SeppeDeWinter** [comment, marked as answer]
Hi @tu4n-p 

I'm still not entirely sure what you mean exactly? 
Do you mean like this: https://pycistopic.readthedocs.io/en/latest/Single_sample_workflow-RTD.html#12.-Gene-activity ?

Best,
Seppe

**Comment by tu4n-p** [comment]
Yes! That's what I meant. Sorry about the wording.

**Comment by SeppeDeWinter** [comment]
It is possible to show these values on the same UMAP, however we don't directly have a function for it. 

It should be trivial however to write some custom code to show this. I would suggest having a look at this function for reference: https://github.com/aertslab/scenicplus/blob/0173233469e045ebff308cc52c6cd2e7ec991206/src/scenicplus/dimensionality_reduction.py#L717

Hope this helps.

best,

Seppe

**Comment by tu4n-p** [comment]
Perfect. Thanks for the rapid response!
